templates/aws/cluster_nodes/terraform/pools/main.tf

```diff
@@ -147,20 +161,48 @@ resource "aws_lb_target_group_attachment" "aws_internal_tg_attachment_443_server
   port             = 443
 }
 
+resource "aws_lb_target_group_attachment" "aws_internal_tg_attachment_6443_server" {
+  count = var.airgap_setup ? var.server_count : 0
+  target_group_arn = aws_lb_target_group.aws_internal_tg_6443[0].arn
+  target_id        = aws_instance.server[count.index].id
+  port             = 6443
+}
+
+resource "aws_lb_target_group_attachment" "aws_internal_tg_attachment_9345_server" {
+  count = var.airgap_setup ? var.server_count : 0
+  target_group_arn = aws_lb_target_group.aws_internal_tg_9345[0].arn
+  target_id        = aws_instance.server[count.index].id
+  port             = 9345
+}
+
 resource "aws_lb_target_group_attachment" "aws_tg_attachment_80" {
-  count = var.agent_count
+  count = var.airgap_setup ? var.agent_count : 0
   target_group_arn = aws_lb_target_group.aws_tg_80.arn
   target_id        = aws_instance.agent[count.index].id
   port             = 80
 }
 
 resource "aws_lb_target_group_attachment" "aws_tg_attachment_443" {
-  count = var.agent_count
+  count = var.airgap_setup ? var.agent_count : 0
```

Comment:
Don't we want a target group for this, even when airgap is not enabled? same comment for above tg_80

Reply:
Sorry for the confusion this PR is based of this one: https://github.com/rancherlabs/corral-packages/pull/10
 can you write the comments there?